Client/WebAuthN/U2FExtensions.swift

```diff
@@ -901,60 +1076,92 @@ class U2FExtensions: NSObject {
-                    self.sendFIDOAuthenticationError(handle: handle, requestId: requestId, errorCode: U2FErrorCodes.other_error)
-                    return
-                }
-                guard let signature = response?.signature.websafeBase64String() else {
-                    self.sendFIDOAuthenticationError(handle: handle, requestId: requestId, errorCode: U2FErrorCodes.other_error)
                     return
                 }
-                guard let clientData = response?.clientData.websafeBase64String() else {
+            
+                guard let u2fservice = self.keyU2fService else {
                     self.sendFIDOAuthenticationError(handle: handle, requestId: requestId, errorCode: U2FErrorCodes.other_error)
                     return
                 }
-                
-                self.fidoRequests[handle] = authSuccess
-                
-                self.cleanupFIDOAuthentication(handle: handle)
-                if requestId >= 0 {
-                    ensureMainThread {
-                        self.tab?.webView?.evaluateJavaScript("u2f.postLowLevelSign(\(requestId), \(true), '\(keyHandle)', '\(signature)', '\(clientData)', \(defaultErrorCode), '')", completionHandler: { _, error in
-                            if error != nil {
-                                let errorDescription = error?.localizedDescription ?? U2FErrorMessages.ErrorAuthentication.rawValue
-                                log.error(errorDescription)
-                            }
-                    }) }
-                    return
-                }
 
-                ensureMainThread {
-                    self.tab?.webView?.evaluateJavaScript("u2f.postSign('\(handle)', \(true), '\(keyHandle)', '\(signature)', '\(clientData)', \(defaultErrorCode), '')", completionHandler: { _, error in
-                        if error != nil {
-                            let errorDescription = error?.localizedDescription ?? U2FErrorMessages.ErrorAuthentication.rawValue
-                            log.error(errorDescription)
+                u2fservice.execute(signRequest) { [weak self] response, error in
+                    guard let self = self else {
+                        log.error(U2FErrorMessages.ErrorAuthentication.rawValue)
+                        return
+                    }
+                    
+                    count -= 1
+                    self.fidoRequests[handle] = count
+                    
+                    guard error == nil else {
+                        if count == 0 {
+                            let errorMessage = error?.localizedDescription ?? Strings.U2FAuthenticationError
+                            self.sendFIDOAuthenticationError(handle: handle, requestId: requestId, errorCode: U2FErrorCodes.other_error, errorMessage: errorMessage)
+                        }
+                        return
+                    }
+                    guard let keyHandle = response?.keyHandle else {
```

Comment:
`keyHandle`, `signature` and `clientData` can go under the same `guard` condition, just split it with new lines to avoid one long line